src/main/java/aquality/selenium/elements/interfaces/IElementFactory.java

```diff
@@ -339,4 +339,121 @@ default ITextBox getTextBox(By locator, String name, ElementState state) {
+     * @return List of child elements.
+     */
+    default <T extends IElement> List<T> findChildElements(IElement parentElement, By childLoc,
+                                                           ElementType type, ElementState state) {
+        return findChildElements(parentElement, childLoc, type, ElementsCount.ANY, state);
+    }
+
+    /**
+     * Finds child elements by their locator relative to parent element.
+     *
+     * @param parentElement Parent element for relative search of child elements.
+     * @param childLoc      Locator of child elements relative to its parent.
+     * @param type          Type of the element to be obtained
+     * @param count         Expected number of elements that have to be found (zero, more then zero, any).
+     * @param state         Visibility state of child elements.
+     * @return List of child elements.
+     */
+    default <T extends IElement> List<T> findChildElements(IElement parentElement, By childLoc, ElementType type,
+                                                           ElementsCount count, ElementState state) {
+        return findChildElements(parentElement, childLoc, null, type, count, state);
+    }
+
+    /**
+     * Finds displayed child elements by their locator relative to parent element.
+     *
+     * @param parentElement Parent element for relative search of child elements.
+     * @param childLoc      Locator of child elements relative to its parent.
+     * @param name          Child elements name.
+     * @param type          Type of the element to be obtained
+     * @return List of child elements.
+     */
+    default <T extends IElement> List<T> findChildElements(IElement parentElement, By childLoc,
+                                                           String name, ElementType type) {
+        return findChildElements(parentElement, childLoc, name, type, ElementsCount.ANY);
+    }
+
+    /**
+     * Finds displayed child elements by their locator relative to parent element.
+     *
+     * @param parentElement Parent element for relative search of child elements.
+     * @param childLoc      Locator of child elements relative to its parent.
+     * @param name          Child elements name.
+     * @param type          Type of the element to be obtained
+     * @param count         Expected number of elements that have to be found (zero, more then zero, any).
+     * @return List of child elements.
+     */
+    default <T extends IElement> List<T> findChildElements(IElement parentElement, By childLoc, String name,
+                                                           ElementType type, ElementsCount count) {
+        return findChildElements(parentElement, childLoc, name, type, count, ElementState.DISPLAYED);
+    }
+
+    /**
+     * Finds child elements by their locator relative to parent element.
+     *
+     * @param parentElement Parent element for relative search of child elements.
```

Comment:
add @param <T> Type of the target elements. please and for every method above

Reply:
fixed